index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio Homepage</title>
+</head>
+
+<body>
+  <div id="toolbar">
+    <div id="logoPlaceholder">
+      <p id="logoText">Welcome</p>
+    </div>
+    <input id="headerButton" type="image" src="Images/Enabled.png" alt="Menu">
+  </div>
+  <div id="headline">
+    <div class="primaryText" id="primaryText">
+      <p class="primaryText" id="primaryName">
+        Hey there. I'm Ricardo
+      </p>
+      <p class="primaryText" id="primaryJob">
+        I'm a software developer
+      </p>

```

Comment:
Since this is a subtitle, it would be best to make use of one of the sub-headings tags `h2, h3 ... h6` in order to follow the hierarchal nature of headings.